privacy.html

```diff
@@ -119,4 +119,31 @@
 	#privacy_optin, #privacy_optout {
 		-webkit-appearance: auto;
 	}
+
+	.ak-privacy.gdpr-hidden {
+    	display: none !important;
+  	}
 </style>
+
+<script>
+  jQuery(document).ready(function($){
```

Comment:
should we condition this to only happen in on pages where the after action step is set to `flow`?